tests/behat/features/web/translation/show_custom_translation.feature

```diff
@@ -0,0 +1,37 @@
+@web @project_page
+Feature: Projects should show a previously defined custom translation for user's language
+
+  Background:
+    Given there are users:
+      | id | name      |
+      | 1  | Catrobat  |
+    And there are projects:
+      | id | name      | owned by  | description    | credit     |
+      | 1  | project 1 | Catrobat  | my description | my credits |
+    And there are project custom translations:
+      | project_id | language | name            | description            | credit             |
+      | 1          | de       | translated name | translated description | translated credits |
+
+  Scenario: Viewing a previously entered custom description translation
+    Given I am on "/app/project/1"
+    Then the selected language should be "English"
+    And I wait for the page to be loaded
+    And I should see "my description"
+    But I should not see "translated description"
```

Comment:
I think we should use a separate scenario for "not showing custom translation". Also I think we could merge the two scenarios on "description" and "credit" into one scenario. So all in all, one scenario for showing custom translation, one for not showing any.

Reply:
Update to two tests, one showing a translation and one not showing the translation